backend/src/game/Dealer.ts

```diff
@@ -46,6 +47,10 @@ export class Dealer {
     });
   }
 
+  public static foldCards(player: Player) {
+    player.foldCards();
+  }
```

Comment:
I think this is where the logic from `PokerTable.foldPlayer` should live and so we should call `Dealer.foldPlayer(pokerTableName, userId)` from the `GamesFoldHandler`

Think of the `Dealer` = `GameService` so "games.fold`

Reply:
I updated the function to PokerTable.foldCards. Do you think we can reuse foldCards in multiple classes?